### ISSUE TITLE
null version for dependencies

### ISSUE THREAD

**Issue by KutnerBitsrc** [teambit/bit#1955]
<img width="460" alt="Screen Shot 2019-08-25 at 12 15 06" src="https://user-images.githubusercontent.com/46922515/63648032-7531c900-c732-11e9-81c8-a71264d2b9b7.png">

I am getting ```null``` version for dependencies in the component I just imported.
I ejected it from my project, but I got errors, so I imported it back.

I'm not sure how to reproduce this, it doesn't happen when I run bit import in other project.

### Specifications

- Bit version: 14.2.5
- Node version: 12.1.0
- yarn version: 1.16.0
- Platform: macOs 10.14.6

**Comment by KutnerBitsrc** [comment]
ok I tried to reproduce it again, and it's not happening.

Scss dependencies are not detected correctly. Getting this error:
```
     > comp.card ...  issues found       
       missing packages dependencies (use your package manager to make sure all package dependencies are installed): 
          styles.scss -> ~@bit
```

I manually ```npm install```ed '~@bit/kutner.experiments.theme.neon-colors', and it seemed to work, but only react appeared in ```bit show```.

I think I might have killed the process while I was exporting _comp/card_. So maybe it was a race condition?

Finally, I made a workaround by ejecting and importing the component. It fixed the dependencies.
